<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameDialog</class>
 <widget class="QDialog" name="RenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="prefixButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="prefixEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Prefix</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="nameButton">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="placeholderText">
      <string>Name_####</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="suffixButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="suffixEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Suffix</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="numberButton">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="numberSpinBox">
     <property name="prefix">
      <string>Number from: </string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Include one or several '#'
where the numbers must be.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RenameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RenameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>numberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prefixButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>prefixEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>nameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>suffixButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>suffixEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
